playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/post.yml

```diff
@@ -37,7 +37,7 @@
   - name: Update router image to current version
     when: all_routers.rc == 0
     command: >
-      {{ oc_cmd }} patch dc/{{ item['labels']['deploymentconfig'] }} -p
+      {{ oc_cmd }} patch dc/{{ item['labels']['deploymentconfig'] }} -n {{ item['namespace'] }} -p
```

Comment:
@dgoodwin this is probably a pretty big issue considering Cloud Forms integration relies on a router that is not in the default namespace.